src/shared/components/overlay/modal/GeneralModal.tsx

```
@@ -0,0 +1,73 @@
+  confirmVariant,
+  showCreditChip = false,
+  creditCount = 0,
+  maxCredit = 0,
+}: GeneralModalProps) => {
+  return (
+    <div className={styles.backdrop} onClick={onClose}>
+      <div
+        className={styles.container}
+        role="dialog"
+        aria-modal="true"
+        onClick={(e) => e.stopPropagation()}
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)